src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -187,7 +186,7 @@ Amplify Studio allows you create auth resources, set up authorization rules, imp
 <Block name="Existing Resources">
 
 Existing Authentication resources from AWS (e.g. Amazon Cognito UserPools or Identity Pools) can be used with the Amplify Libraries by calling the `Amplify.configure()` method.
-
+ 
```

Comment:
I added this here b/c we switched the order b/w installing the libraries from the current getting started page. Can/should we have the install amplify libraries prompt here so those using existing resources can follow or do we assume they already did this step?

Reply:
I commented the same somewhere else - ideally, we'll want to give them the steps to install the library for exsiting resources as well.